### DISCUSSION TITLE
【1966720】致命公司 清单+密钥

### DISCUSSION THREAD

**Discussion by Swor-dart** [pjy612/SteamManifestCache#102] [Share]
appid:1966720
keys:
```
1966721;b79233b678dae6d3f928ff9c6c7674d559427776a1337dee238773a4fec60949
```
[致命公司.zip](https://github.com/pjy612/SteamManifestCache/files/13726402/default.zip)
内测码：93B96C7014454872

**Comment by pjy612** [comment]
漏了信息

**Reply by Swor-dart** [reply]
漏了啥啊？appid和keys，清单密钥也上传zip了。
邮箱吗？zhclvchina@gmail.com

**Reply by pjy612** [reply]
是啊，不然怎么发给你 233，查收
